common/test/acceptance/pages/studio/home.py

```diff
@@ -0,0 +1,18 @@
+"""
+Home page for Studio when logged in.
+"""
+from bok_choy.page_object import PageObject
+
+from common.test.acceptance.pages.studio import BASE_URL
+
+
+class HomePage(PageObject):
```

Comment:
Is there a difference between this and common/test/acceptance/pages/studio/index.py? Just the URL? is there a way to combine the two?